docker-compose.test.yml

```diff
@@ -0,0 +1,55 @@
+version: "3.7"
+
+services:
+  postgres:
+    image: postgres:12.6
+    ports:
+      - 5432
+    environment:
+      POSTGRES_USER: test
+      POSTGRES_PASSWORD: test
+      POSTGRES_DB: test
+    volumes:
+      - postgres-data:/var/lib/postgresql/data
+
+  ora11:
+    image: wnameless/oracle-xe-11g-r2
+    ports:
+      - 1521
+    environment:
+      - ORACLE_ALLOW_REMOTE=true
+
+  river:
+    build:
+      context: .
+      dockerfile: Dockerfile.test
+    entrypoint: ""
+    # keep container alive for test commands with `docker-compose exec`
+    command: tail -F anything
```

Comment:
> et l'un devra mourir de la main de l'autre car aucun d'eux ne peut vivre tant que l'autre survit
Sibylle Trelawney

![media](https://media.giphy.com/media/aU4jau5ql99EA/giphy.gif)

Reply:
I don't think I can remove one of them...
`entrypoint: ""` is here to override the django entrypoint
`command: tail -F anything` is here to keep the container alive

Reply:
ok :+1: